src/cat-data-service/src/service/common/objects/voter.rs

```diff
@@ -0,0 +1,44 @@
+use super::voter_info::VoterInfo;
```

Comment:
By breaking into multiple files its a lot easier to review.
Thankyou.

It is clear here that this isn;t a voter, but the voters latest valid registration.
I suggest this should be called "voter_registration.rs"

```suggestion
//! Define information about the Voters Registration.
use super::voter_info::VoterInfo;
```